src/services/rollouts.js

```diff
@@ -122,4 +125,144 @@ module.exports = class RolloutsHelper {
 			throw error
 		}
 	}
+	/**
+	 * Rollout List
+	 * @method
+	 * @name list
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - List of rollouts
+	 */
+	static async list(organization_id, loggedInUserId, queryParams, searchText = '', page, limit) {
+		try {
+			let result = {
+				data: [],
+				count: 0,
+			}
+			let filters = {
+				organization_id,
+				user_id: loggedInUserId,
+			}
+
+			if (searchText && searchText != '') {
+				filters.title = {
+					[Op.iLike]: '%' + searchText + '%',
+				}
+			}
+
+			if (queryParams.type && queryParams.type != '') {
+				filters.type = queryParams.type
+			}
+
+			if (queryParams.status && queryParams.status != '') {
+				filters.status = {
+					[Op.in]: queryParams.status.trim().split(','),
+				}
+			}
+
+			const sort = await this.constructSortOptions(queryParams?.sort_by, queryParams?.sort_order)
+
+			const rolloutList = await rolloutQueries.findAllAndCount(
```

Comment:
@adithyadinesh0412  do we need to show the status as inactive in the listing ?

Reply:
That was what was concluded right? @priyanka-TL . To show the Inactive rollouts. please correct me if im wrong.